sdk/data/azcosmos/cosmos_container.go

```diff
@@ -399,6 +401,61 @@ func (c *ContainerClient) DeleteItem(
 	return newItemResponse(azResponse)
 }
 
+// QueryItems executes a single partition query in a Cosmos container.
+// ctx - The context for the request.
+// query - The SQL query to execute.
+// partitionKey - The partition key to scope the query on.
+// o - Options for the operation.
+func (c *ContainerClient) QueryItems(query string, partitionKey PartitionKey, o *QueryOptions) *runtime.Pager[QueryItemsResponse] {
```

Comment:
We've recently updated our guidelines for naming pager APIs.  This one would be `NewQueryItemsPager()`

Reply:
Done, renamed to `NewQueryItemsPager`

Reply:
APIView diff also updated